src/lxc/network.c

```diff
@@ -3508,13 +3508,6 @@ static int lxc_setup_netdev_in_child_namespaces(struct lxc_netdev *netdev)
 		if (netdev->type == LXC_NET_NONE)
 			return 0;
 
-		if (netdev->type != LXC_NET_VETH) {
-			net_type_name = lxc_net_type_to_str(netdev->type);
```

Comment:
Removing this turned net_type_name into an unused variable. Detected by travis.

Reply:
On it, thanks!